packages/react-components/theme-designer/src/components/Sidebar/Sidebar.tsx

```diff
@@ -8,13 +10,84 @@ export interface SidebarProps {
 const useStyles = makeStyles({
   root: {
     display: 'flex',
+    flexDirection: 'column',
     alignItems: 'center',
-    justifyContent: 'center',
+    justifyContent: 'flex-start',
     ...shorthands.borderRight('1px', 'solid', '#D1D1D1'),
+    ...shorthands.gap(tokens.spacingVerticalXXL),
```

Comment:
I believe this has been put on hold but we've been moving away from using `flex-gap` in this repo because of browser support requirements. @TristanWatanabe will know the latest.

I think we're fine to use it here as this is an app but just wanted to highlight it as there's been ongoing discussion about this CSS property.

Reply:
are we applying the same support matrix for dev tooling?

Reply:
Should be fine to use this here since it's an app that wont be directly consumed by partners :). The proposed browser support matrix should only apply to v9 controls.

Reply:
> are we applying the same support matrix for dev tooling? i sure hope not :)

> The proposed browser support matrix should only apply to v9 controls.

100% agreed. At the same time any code in our repo will be looked at as an example of the "right way" to do things.